.github/workflows/test.yml

```diff
@@ -114,6 +115,11 @@ jobs:
             python: "3.11-dev"
             os: ubuntu-latest
             tox_env: "py311"
+            use_coverage: true
+          - name: "ubuntu-py311-exceptiongroup"
+            python: "3.11-dev"
+            os: ubuntu-latest
+            tox_env: "py311-exceptiongroup"
```

Comment:
Don't we want to test this in Python < 3.11, so as to test our `exceptiongroup` external package support? IIUC this is testing in Python 3.11 using the native `ExceptionGroup`.

Reply:
This environment is for testing that 3.11 works when tested code raises `exceptiongroup.ExceptionGroup` and it's caught as a native `ExceptionGroup`. The standard `py37` checks that `exceptiongroup.ExceptionGroup` works, and `py311` that native `ExceptionGroup` works when `exceptiongroup` is not installed.

Reply:
Ahh right, thanks for the clarification!

Reply:
though since the implementation has ended up this simple, running CI for a single test that's pretty much only gonna break if `exceptiongroup` breaks is maybe a bit silly

Reply:
Indeed, agree.

Feel free to drop this then. 👍